README.md

```diff
@@ -80,6 +80,20 @@ $ docker-compose run --rm api rake db:create db:schema:load linksf:import
 $ docker-compose run -e STAGING_DB_PASSWORD=<...> --rm api rake db:setup db:import_staging
 ```
 
+### Running tools from the terminal
+Due to the DB and API being contained inside Docker containers, running ruby/rails specific tools without having to ssh into the containers involves specific steps
+#### Running the rails console from the terminal
+```sh
+# Start both the DB and API
+$ cd PATH/TO/askdarcel-api
+$ docker-compose run --rm api rails console
+```
+
+#### Running byebug from the terminal
+```sh
+# Start both the DB and API
+$ docker attach $(docker ps -aqf "name=askdarcel-api_api")
```

Comment:
It might be worth describing this in a lot more detail. For people who aren't familiar with byebug, this doesn't actually teach anyone how to start byebug. What this command really does is attach your TTY and stdin to an already running Docker container which is assumed to be running the Rails web server. The comment above this is actually incorrect; this command will fail if the API server is not already running.

Reply:
I have added more information; let me know if you want me to add more.